config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -599,7 +599,7 @@ presubmits:
     cluster: k8s-infra-prow-build
     always_run: true
     branches:
-      - main
+      # - main
```

Comment:
Interestingly we don't list `release-3.5` below, do you remember why it's omitted @ivanvc?

Reply:
We split the stable branches migration between Joshua (3.4) and Abdur (3.5). They have different availability/speed. Eventually we should have all of the stable branches here. 

Side note: It has been challenging to work on the migration because I didn't own the ticket, and I can't update the completion tracking ref:
https://github.com/kubernetes/test-infra/issues/32754